Example/Core/Tests/FIROptionsTest.m

```diff
@@ -584,4 +585,16 @@ - (void)testVersionConsistency {
   XCTAssertEqualObjects(str, [NSString stringWithUTF8String:(const char *)FIRCoreVersionString]);
 }
 
+// Repeat test with more Objective C.
```

Comment:
Note: this will also break when Firebase major hits 10.

Reply:
Added 2nd comment.

Reply:
Ahh didn't notice the `tests` before - thanks for adding a second comment here.